src/toil/common.py

```diff
@@ -413,6 +414,10 @@ def _addOptionFn(*name, **kwargs):
                       "missing preemptable nodes with a non-preemptable one. A value of 1.0 "
                       "replaces every missing pre-emptable node with a non-preemptable one." %
                       config.preemptableCompensation))
+    addOptionFn("--workerDiskSize", dest="workerRootVolSize", default=None,
```

Comment:
What do you think about ``--nodeStorage`` instead? Also it's generally better to keep a consistent name between the argument and the 'dest' variable.